datalad_next/constraints/parameter.py

```diff
@@ -258,7 +270,9 @@ def joint_validation(self, params: Dict) -> Dict:
     def __call__(self, kwargs, at_default=None) -> Dict:
         validated = {}
         for argname, arg in kwargs.items():
-            if at_default and argname in at_default:
+            if at_default \
+                    and argname not in self._validate_defaults \
+                    and argname in at_default:
```

Comment:
I can see that. I will leave it unchanged for now (to avoid yet another CI run). I should be working on general docs on how this validation works at all. This is likely a better place for this kind of info